tmp/addons/.gitignore

```diff
@@ -1 +0,0 @@
-*
```

Comment:
there was some problem with the jetpack file on my mac earlier on ...i deleted some files then